data/icon-overlay.js

```diff
@@ -0,0 +1,90 @@
+const host = window.location.host;
+
+// YOUTUBE HOME PAGE
+// TODO: listen for infinite scrollesque updates and apply this again
+// if necessary
+const ytHomeContainers = Array.from(document.querySelectorAll('#feed .yt-lockup-thumbnail'));
+if ((host.indexOf('youtube.com') > -1) && ytHomeContainers.length) {
+  ytHomeContainers.forEach(el => {
+    el.classList.add('minvid__overlay__wrapper');
+    const tmp = getTemplate();
+    tmp.addEventListener('click', function(ev) {
+      evNoop(ev);
+      const partialUrl = el.querySelector('.yt-uix-sessionlink').getAttribute('href');
+      if (partialUrl.length) {
+        self.port.emit('launch', {
+          url: 'https://youtube.com' + partialUrl,
+          domain: host
+        });
+      } else console.error('Error parsing url from YT home page', partialUrl); // eslint-disable-line no-console
```

Comment:
**PROTIP:** ESLint has a [`curly`](http://eslint.org/docs/rules/curly) for this very thing:

```sh
$ eslint .

/Users/pdehaan/dev/github/mozilla/min-vid/app.js
  135:5  error  Expected { after 'if' condition  curly

/Users/pdehaan/dev/github/mozilla/min-vid/data/resize-listener.js
   7:7  error  Expected { after 'if' condition  curly
  12:5  error  Expected { after 'if' condition  curly

/Users/pdehaan/dev/github/mozilla/min-vid/index.js
   30:5  error  Expected { after 'if' condition  curly
   31:5  error  Expected { after 'else'          curly
   79:7  error  Expected { after 'if' condition  curly
   97:7  error  Expected { after 'if' condition  curly
  113:7  error  Expected { after 'if' condition  curly

✖ 8 problems (8 errors, 0 warnings)
```

---

PR incoming!